src/redux/reducers/index.ts

```diff
@@ -4,17 +4,20 @@ import { gameReducer, GameState } from './game';
 import { opponentReducer, OpponentState } from './opponents';
 import { loginReducer, LoginState } from './login';
 import { WalletState, walletStateReducer } from '../../wallet/redux/reducers/wallet-state';
+import { drizzleReducers } from 'drizzle';
 
 export interface ApplicationState {
   game: GameState;
   opponents: OpponentState;
   login: LoginState;
   wallet: WalletState;
+  contracts:any;
```

Comment:
`any[]` is slightly better, I suppose? Is there any way to define a `Contract` type?